support-frontend/assets/helpers/campaigns/campaigns.tsx

```diff
@@ -28,7 +28,7 @@ export type CampaignTickerSettings = Omit<TickerSettings, 'tickerData'> & {
 	id: string;
 };
 
-export type CampaignSettings = {
+type CampaignSettings = {
```

Comment:
No longer used by the landing page - we can probably remove this file later

Reply:
I've never really understood the campaign feature...are we going to retire it?

Reply:
yes I think so, it'll eventually be replaced by the configuration from the server